src/client/pps/pps.proto

```diff
@@ -70,9 +70,11 @@ message PipelineInput {
 }
 
 enum PipelineState {
-    // After this pipeline is created, but before it's picked up by any
-    // pachd node.
-    PIPELINE_STARTING = 0;
+    // When the pipeline is not ready to be triggered by commits.
+    // This happens when either 1) a pipeline has been created but not yet picked
+    // up by a PPS server, or 2) the pipeline does not have any inputs and is meant
+    // to be triggered manually
+    PIPELINE_IDLE = 0;
```

Comment:
While we're changing names it's a little weird to me that we have `JOB_FAILURE` and `PIPELINE_FAILED`. Could we change to `PIPELINE_FAILURE` to match?